feature/galaxy/src/main/java/com/revenuecat/purchases/galaxy/GalaxyConfiguration.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * This class is in Java because there is a weird issue when written in Kotlin. The class
+ * can't be used as a PurchasesConfiguration.Builder from a Java class when written in Kotlin.
+ *
+ * If written in Kotlin this wouldn't work:
+ *
+ * PurchasesConfiguration.Builder builder = new GalaxyConfiguration.Builder(this, "");
+ *
+ * And this either:
+ *
+ * GalaxyConfiguration.Builder builder = new GalaxyConfiguration.Builder(this, "");
+ * builder.build();
+ */
+package com.revenuecat.purchases.galaxy;
+
+import android.content.Context;
+
+import androidx.annotation.NonNull;
+
+import com.revenuecat.purchases.PurchasesConfiguration;
+import com.revenuecat.purchases.Store;
+
+/**
+ * Holds parameters to initialize the SDK for the Galaxy Store. Create an instance of this class using the [Builder]
+ * and pass it to [Purchases.configure].
+ */
+public final class GalaxyConfiguration extends PurchasesConfiguration {
```

Comment:
Should we add API tests for this as well? Alternatively, we could mark these public APIs as experimental, but since we're merging to an integration branch, I think it's ok to add that later only if needed.